allo/allo.py

```diff
@@ -1,3 +1,326 @@
+# -*- coding: utf-8 -*-
+"""
+This file serve as a starting point for a Python console script.
+"""
+import random
+
+from allo.room.room import Office, LivingSpace
+from allo.person.person import Fellow, Staff
+
+
 class Dojo(object):
-    pass
+    """
+    Class Dojo the main entry point to a110-cli
+    """
+    def __init__(self):
```

Comment:
why all these unnecessary data structures to store the same stuff? You only need to store your records once..